UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeProperty.java

```diff
@@ -268,6 +268,10 @@ public int getMaxWidth(boolean getShortest) {
             return maxFirstValueAliasWidth;
         return maxValueWidth;
     }
+    
+    public final UnicodeSet getBoolean() {
```

Comment:
Seems like a misnomer. How about   getYesSet()   ?

Reply:
Changing to getTrueSet (and checking for only boolean properties)